rep-2000.rst

```diff
@@ -709,6 +709,121 @@ Package manager use for dependencies:
++--------------+-----------------------+-------------------------+----------------------+--------------------+----------------+
+|    arm64     |   Tier 1 [d][a][s]    |                         |                      |     Tier 3 [s]     |   Tier 3 [s]   |
++--------------+-----------------------+-------------------------+----------------------+--------------------+----------------+
+|    arm32     |     Tier 3 [s]        |                         |                      |     Tier 3 [s]     |   Tier 3 [s]   |
++--------------+-----------------------+-------------------------+----------------------+--------------------+----------------+
+
+The following indicators show what delivery mechanisms are available for each platform.
+
+" [d] " Debian packages will be provided for this platform for packages submitted to the rosdistro.
+
+" [a] " Binary releases are provided as a single archive per platform containing all packages in the Eloquent ROS 2 repos file [6]_.
+
+" [s] " Compilation from source.
+
+Middleware Implementation Support:
+
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| Middleware Library       | Middleware Provider | Support Level | Platforms                   | Architectures                        |
++==========================+=====================+===============+=============================+======================================+
+| rmw_fastrtps_cpp         | eProsima Fast-RTPS  | Tier 1        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_connext_cpp          | RTI Connext         | Tier 1        | All Platforms except Debian | All Architectures except arm64/arm32 |
+|                          |                     |               | and OpenEmbedded            |                                      |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_cyclonedds_cpp       | Eclipse Cyclone DDS | Tier 1        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-RTPS  | Tier 2        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minimum language requirements:
+
+- C++14
```

Comment:
```suggestion
- C++17
```

Building off from [this conversation](https://github.com/ros-infrastructure/rep/pull/217#discussion_r356796304) from Foxy, we should reconsider whether it is time to move to C++17.

Reply:
Given [this comment](https://github.com/ros-infrastructure/rep/pull/291#issuecomment-747511683), let's go ahead with C++17 support.